platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/JasperDbStatistics.java

```diff
@@ -51,27 +51,27 @@ public class JasperDbStatistics {
     private SpeedometerMetric leafByKeyReadsPerSecond;
     private SpeedometerMetric leafByPathReadsPerSecond;
 
-    private IntegerGauge internalHashesStoreFileCount;
-    private DoubleGauge internalHashesStoreTotalFileSizeInMB;
+    private IntegerGauge hashesStoreFileCount;
+    private DoubleGauge hashesStoreTotalFileSizeInMB;
 
     private IntegerGauge leafKeyToPathStoreFileCount;
 
     private DoubleGauge leafKeyToPathStoreTotalFileSizeInMB;
 
-    private IntegerGauge leafPathToHashKeyValueStoreFileCount;
+    private IntegerGauge leafPathToKeyValueStoreFileCount;
```

Comment:
To be consistent with `hashesStoreFileCount` above, this field should be named `leavesStoreFileCount`. Alternatively, use `pathToHashStoreFileCount` and `pathToKeyValueStoreFileCount`.

Reply:
Done.